Frontend/src/app/auth/pages/login/login.component.html

```diff
@@ -1,3 +1,43 @@
-<p>
-  login works!
-</p>
+<nb-layout>
```

Comment:
Since nb-layout is in the app-component, you are able to remove it from all pages. you can remove the nb-layout, nb-layout-header, as well as the nb-layout-column components. the only thing that should be left is the contents inside the nb-layout-column.

Reply:
nb-layout removed